scripts/azureml-assets/azureml/assets/environment/build.py

```diff
@@ -62,6 +62,11 @@ def create_acr_task(image_name: str,
             'id': 'test',
             'cmd': f"$Registry/{image_name} {test_command}"
         })
+        task['steps'].append({
+            'id': 'scan',
+            'cmd': "wget https://github.com/aquasecurity/trivy/releases/download/v0.42.1/trivy_0.42.1_Linux-64bit.deb && dpkg -i trivy_0.42.1_Linux-64bit.deb && trivy fs --scanners vuln /",
+            'ignoreErrors': True

```

Comment:
I think you'll need the name of the image to run the command on and let's be explicit about the bool we pass for `ignoreErrors`.
```suggestion
            'cmd': f"$Registry/{image_name} wget https://github.com/aquasecurity/trivy/releases/download/v0.42.1/trivy_0.42.1_Linux-64bit.deb && dpkg -i trivy_0.42.1_Linux-64bit.deb && trivy fs --scanners vuln /",
            'ignoreErrors': 'true'
```

Reply:
You'll probably need to break this over multiple lines to make flake8 happy.